### DISCUSSION TITLE
有没有视频介绍或教学内容的？

### DISCUSSION THREAD

**Discussion by wei1224hf** [ZhuYanzhen1/miniFOC#5] [General]
有没有这个产品的油管或B站视频可以看一下的？

**Comment by ZhuYanzhen1** [comment]
> 
> 
> 有没有这个产品的油管或B站视频可以看一下的？

还没有搞，最近学业繁忙example还没来得及做。不过你可以在这个帖子里面找到相关的一些视频：https://whycan.com/t_7119.html
